File: C/1043.c
#include <stdio.h>

int main(void) {
  double A;
  double B;
  double C;
  double res;

  scanf("%lf", &A);
  scanf("%lf", &B);
  scanf("%lf", &C);

  if(A<B+C && B<A+C && C<A+B){
    res = A + B + C;
    printf("Perimetro = %.1lf\n", res);
  }else{
    res = ((A+B)*C)/2;
    printf("Area = %.1lf\n", res);
  }

  return 0;
}

File: Python/1043.py
a, b, c = input().split()

a = float(a)
b = float(b)
c = float(c)

if (a < b+c) and (b < a+c) and (c < a+b):
    resposta = a + b + c
    print('Perimetro = {:.1f}' .format(resposta))
else:
    resposta = ((a+b)*c)/2
    print('Area = {:.1f}' .format(resposta))